recipes/ng_nodes_certificates.rb

```diff
@@ -23,7 +23,7 @@
 end
 
 execute 'Wait for API to become available' do
-  command "[[ $(curl --silent --tlsv1.2 --max-time 2 #{node['cookbook-openshift3']['openshift_master_api_url']}/healthz/ready --cacert #{node['cookbook-openshift3']['master_certs_generated_certs_dir']}/ca.crt --cacert #{node['cookbook-openshift3']['master_certs_generated_certs_dir']}/ca-bundle.crt) =~ \"ok\" ]]"
+  command "[[ $(curl --silent --tlsv1.2 --max-time 2 #{node['cookbook-openshift3']['openshift_master_loopback_api_url']}/healthz/ready --cacert #{node['cookbook-openshift3']['master_certs_generated_certs_dir']}/ca.crt --cacert #{node['cookbook-openshift3']['master_certs_generated_certs_dir']}/ca-bundle.crt) =~ \"ok\" ]]"
```

Comment:
This one should not be changed as it belongs to the certificate server.
The certificate server has not necessarily got master running services and therefore changing it to loopback_api is likely to fail...
Can you please leave this one

Reply:
Hello @IshentRas if the nodes certificates can be generated on a server which is not master, then indeed we cannot use the loopback master api url.

Do you have a solution to wait for API to be available if we cannot assume that the certificate on the master api URL has been issued by the kubernetes ca (in the case of namedCertificates) ? Using the IPaddress of the first master may work, but then we have to be careful when using http/https proxies.

Reply:
I think what you have proposed is correct. I will just change the logic for:
command: is_master_server ? ".... (With loopback)" : "...(With master_api_url + ignore cert)"

Reply:
OK, can I let you append a commit to this PR?

Reply:
I'll do it :)